### PR TITLE
Armv7 code refactor

### PR DESCRIPTION
**Pull request by disconnect3d**
Removes dead imports, makes code style more consistent and a bit more compliant with pep8 (yeah i seems we don't enforce any particular style, but this seems to be a reasonable default for me).

...lets see if it passes CI checks.

---
This change is [Reviewable](https://reviewable.io/reviews/trailofbits/manticore/749)

### CODE REVIEW

**Comment by mossberg** [comment]
great, we'll get to reviewing this shortly!

**Comment by disconnect3d** [comment]
@mossberg I have rebased this to the latest master so it can be reviewed :P.

File: manticore/core/cpu/arm.py [hunk 1]
```diff
@@ -404,23 +408,23 @@ def setFlags(self, **flags):
...
         # XXX: capstone incorrectly sets .update_flags for adc
         if self.instruction.mnemonic == 'adc':
             return
         for flag, val in self._last_flags.iteritems():
             flag_name = 'APSR_{}'.format(flag)
             self.regfile.write(flag_name, val)
 
-    def _Shift(cpu, value, _type, amount, carry):
-        assert(_type > cs.arm.ARM_SFT_INVALID and _type <= cs.arm.ARM_SFT_RRX_REG)
+    def _shift(cpu, value, _type, amount, carry):
```
Comment (mossberg) [inline manticore/core/cpu/arm.py]: The reason we originally had it named like this was to closely match the arm manual pseudocode; however I think this change is a good thing to make the codebase more pythonic. Can you please add a docstring which states "See Shift() and Shift_C() in the ARM manual"?

image

Comment (disconnect3d) [reply manticore/core/cpu/arm.py]: @mossberg added

**Pull request merged**